docs/Release_Guide/release_steps/coordinated/release_acceptance_testing.rst

```diff
@@ -11,9 +11,9 @@ cycles of the METplus components included in a coordinated release.
   * Navigate to the
     `Release Acceptance Testing <https://github.com/dtcenter/METplus/discussions/categories/release-acceptance-testing>`_
     discussion for this coordinated release.
-  * Review and finalize any entries still marked as **OPEN** or **FAIL** in the **Status**
-    column of the repository testing tables. Ideally, the status for all table entries
-    should be **PASS** for a coordinated release.
+  * Review the repository testing tables for the METplus components within the body of the discussion.
```

Comment:
```suggestion
  * Review the repository testing tables for the METplus components within the body of the discussion by clicking on the dropdown icon "Repository Testing" for the appropriate METplus component to expand the hidden table.
```
I'm adding this because I didn't know there were dropdown menus.  (I thought they were bullet points.) And when I searched on the word "pass" it wasn't found because it was in a hidden table.  That's how I got really confused.  (Sphinx will search dropdown menus for text but apparently GitHub does not.)